test/actions/katello/host/update_system_purpose_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'katello_test_helper'
+
+module Katello::Host
+  class UpdateSystemPurposeTest < ActiveSupport::TestCase
+    include Dynflow::Testing
+    include Support::Actions::Fixtures
+    include FactoryBot::Syntax::Methods
+
+    before :all do
+      User.current = users(:admin)
+      @host = FactoryBot.create(:host, :with_subscription)
+    end
+
+    describe 'update system purpose' do
```

Comment:
I'd really like to see us move away from this spec style of test for consistency and for debugging. For example, you can't use ktest to run just this test in the file, so it can get really cumbersome.

Do you mind reformatting the test into unit style? ie `def test_update_system_purpose` ?

Reply:
Updated